### PR TITLE
Fixes react scripts

### PR DESCRIPTION
**Pull request by PuneetPunamiya**
# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/main/standards.md#principles) (if functionality changed/added)
- [ ] Run API Unit Tests, Lint Checks, API Design, Golden Files with `make api-check`
- [ ] Run UI Unit Tests, Lint Checks with `make ui-check`
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/main/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/main/CONTRIBUTING.md) for more details._

### CODE REVIEW

File: ui/src/containers/Search/Search.test.tsx [hunk 1]
```diff
@@ -40,10 +41,8 @@ describe('search resources', () => {
         return !resources.isLoading;
       },
       () => {
-        const node = component.find(TextInput);
-
         act(() => {
-          node.props().onChange('golang');
+          component.find(TextInput).simulate('change')
```
Comment (vinamra28) [inline ui/src/containers/Search/Search.test.tsx:45]:
```suggestion
          component.find(TextInput).simulate('change');
```

**Review by vinamra28 — COMMENTED** [review]
@PuneetPunamiya, overall PR looks good. Can you please run `npm run lint` once and fix the errors?

File: ui/src/containers/Search/Search.test.tsx (hunk 1 above)
Comment (vinamra28) [reply ui/src/containers/Search/Search.test.tsx]: nope

**Review by vinamra28 — APPROVED** [review]

**Comment by pratap0007** [comment]
docker build is failing, could you please check once ? and pr looks good to me .

**Pull request merged**